### PR TITLE
Update COMPILING.md

### PR DESCRIPTION
**Pull request by Seadragon91**
This updates the file COMPILING.md, changes the name `MCServer` -> `Cuberite`.

### CODE REVIEW

**Comment by Seadragon91** [comment]
In the section Profiling is a info about a file named `profile_run.cmd`, but this file doesn't exists anymore. Should I remove the lines?

**Comment by madmaxoft** [comment]
I think the entire Profiling section is no longer relevant, since MSVC 2013 Community Edition includes the profiler directly within the IDE.

**Comment by Seadragon91** [comment]
I removed the section Profiling and changed the `git clone` lines.

File: COMPILING.md [hunk 1]
```diff
@@ -118,11 +100,11 @@ cmake -DCMAKE_BUILD_TYPE=DEBUG ..
 ```
 NOTE: CMake can generate project files for many different programs, such as Xcode, eclipse, and ninja. To use a different generator, first type `cmake --help`, and at the end, cmake will output the different generators that are available. To specify one, add `-G` followed by the name of the generator, in the `cmake` command. Note that the name is case-sensitive. 
 
-The executable will be built in the `MCServer/MCServer` folder and will be named `MCServer_debug`.
+The executable will be built in the `Cuberite/Server` folder and will be named `Cuberite_debug`.
```
Comment (SphinxC0re) [inline COMPILING.md]: s/Cuberite/cuberite

**Comment by Seadragon91** [comment]
Okay my question. Cuberite oder cuberite?

**Comment by SphinxC0re** [comment]
The executable is named **C**uberite(_debug). The repo folder is named **c**uberite

**Comment by Seadragon91** [comment]
Have changed it.

**Pull request merged**